setup.cfg

```diff
@@ -24,14 +24,14 @@ install_requires =
     nibabel ~= 3.0.0rc1
     nipype >=1.3.1
     nitime
-    niworkflows ~= 1.1.3
+    niworkflows ~= 1.1.5
     numpy
     pandas
     psutil >=5.4
     pybids ~= 0.9.4
     pyyaml
     sdcflows ~=1.1.0
-    smriprep ~= 0.4.2
+    smriprep @ git+https://github.com/poldracklab/smriprep.git@46270e8847f36a2dd677e810162358eb69acb211
```

Comment:
```suggestion
    smriprep ~= 0.5
```

Reply:
0.5.0. the distinction matters with ~=

Reply:
Oh, looks like Oscar already did this. (On my phone...)